opds.py

```diff
@@ -455,21 +456,22 @@ def add_ratings(cls, work, entry):
                 entry.append(cls.rating_tag(type_uri, value))
 
     @classmethod
-    def categories(cls, work, identifier_cutoff=100):
+    def categories(cls, work, policy=None):
         """Send out _all_ categories for the work.
 
         (So long as the category type has a URI associated with it in
         Subject.uri_lookup.)
 
-        :param identifier_cutoff: When calculating related identifiers
-        for this work at a given level, cut off the query after
-        approximately this many results. This will improve
-        performance, at the expense of ignoring classifications for
-        identifiers that are distantly related to the work.
+        :param policy: A PresentationCalculationPolicy to
+           use when deciding how deep to go when finding equivalent
+           identifiers for the work.
         """
+        policy = policy or PresentationCalculationPolicy(
+            equivalent_identifier_cutoff=100
```

Comment:
Or you meant to use that constant here?

Reply:
No, this is OK, but if I reduce the cutoff to 100 everywhere I can get rid of this.